cmd/queue/main.go

```diff
@@ -214,10 +225,12 @@ func handler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, handler http.H
 			return
 		case network.IsKubeletProbe(r):
 			// Do not count health checks for concurrency metrics
-			handler.ServeHTTP(w, r)
+			handler.ServeHTTP(recorder, r.WithContext(reqCtx))
 			return
 		}
-
+		probeSpan.End()
+		reqCtx, proxySpan := trace.StartSpan(r.Context(), "proxy")
```

Comment:
Does this need to have `fullCtx` to nest properly?

Reply:
It doesn't. We won't have a `fullCtx` since we're removing the `queueproxy` span.